core/trino-parser/src/main/java/io/trino/sql/ReservedIdentifiers.java

```diff
@@ -39,8 +39,9 @@
 public final class ReservedIdentifiers
 {
     private static final Pattern IDENTIFIER = Pattern.compile("'([A-Z_]+)'");
-    private static final Pattern TABLE_ROW = Pattern.compile("``([A-Z_]+)``.*");
-    private static final String TABLE_PREFIX = "============================== ";
+    private static final Pattern TABLE_ROW = Pattern.compile("\\| `([A-Z_]+).*");
```

Comment:
Match the closing quote of the keyword, just to be safe and consistent
```suggestion
    private static final Pattern TABLE_ROW = Pattern.compile("\\| `([A-Z_]+)` .*");
```

Reply:
Updated